requests_toolbelt/utils/dump.py

```diff
@@ -115,12 +219,17 @@ def _coerce_to_bytes(data):
 
 
 def dump_response(response, request_prefix=b'< ', response_prefix=b'> ',
-                  data_array=None):
+                  data_array=None, sanitizer=NOOP_SANITIZER):
     """Dump a single request-response cycle's information.
 
     This will take a response object and dump only the data that requests can
     see for that single request-response cycle.
 
+    If the optional `sanitize` parameter is used, it should be an object that
```

Comment:
This should be

```
If the optional ``sanitize`` parameter is used, it should be an object that
```

Note the double backticks. They're important because we're not using markdown.

Reply:
Oops, will fix.